docs/reference/gc-toolkit/onboarding/index.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 0
+---
+
+# Onboarding
+
+This document is your onboarding document to a fresh Guardian Connector instance.
```

Comment:
Maybe add a meta-note about how this is a template to be used to help onboard a community. So that if someone stumbles onto this page in gc-docs, they have the context to understand its purpose and why it is e.g. written in the first person ("I / you" language).

Could use Docusaurus convention like

```
:::note

...

:::
```